onnxscript/tests/function_libs/torch_lib/extra_opinfo.py

```diff
@@ -702,6 +709,7 @@ def sample_inputs_index(op_info, device, dtype, requires_grad, **kwargs):
     index_1d = common_methods_invocations.index_variable(2, s, device=device)
     index_2d = common_methods_invocations.index_variable((s + 1, 2), s, device=device)
     index_3d = common_methods_invocations.index_variable((s + 2, s + 1, 2), s, device=device)
+    index_bool = _index_variable_bool(s, s, device=device)
```

Comment:
Can we break this case out into a different name so we don’t need to add skips below?